ComponentFramework3.0/ComponentFramework/GCamera.cpp

```diff
@@ -0,0 +1,124 @@
+#include "GCamera.h"
+
+using namespace GAME;
+
+
+GCamera::GCamera(const int viewportWidth, const int viewportHeight, const Vec3 pos_)
+{
+	//Camera Parts
+	pos = pos_; //Starting-> Current Position
+	Opos = pos_; //Starting
+	orientation = Vec3(0.0f, 0.0f, 10.0f);
```

Comment:
avoid the use of magic numbers (numbers seemingly picked at random that work) create a constant and refer to that constant instead. some numbers are special like 0, 1 or /2 but for almost any other number please use a constant.

Reply:
Google "code smells" and "magic numbers" if you are unfamiliar with the term.